Makefile

```diff
@@ -133,6 +133,9 @@ test-race:
 test-cover:
 	@go test -mod=readonly -timeout 30m -race -coverprofile=coverage.txt -covermode=atomic -tags='ledger test_ledger_mock' ./...
 
+test-integration:
+	@go test -mod=readonly -tags="cli_test" ./.../testutil/...
+
```

Comment:
I think it's inappropriate to have a `make test-integration` when the tags are `cli_test`.

Reply:
I changed it to `test-cli`.